tests/test_result.py

```diff
@@ -7,31 +7,31 @@
 import pytest
 
 import tox
-from tox.result import ResultLog
+import tox.result
 
 
-@pytest.fixture
-def pkg(tmpdir):
-    p = tmpdir.join("hello-1.0.tar.gz")
-    p.write("whatever")
-    return p
+@pytest.fixture(name="pkg")
+def create_fake_pkg(tmpdir):
+    pkg = tmpdir.join("hello-1.0.tar.gz")
+    pkg.write("whatever")
+    return pkg
 
 
-def test_pre_set_header(pkg):
-    replog = ResultLog()
+def test_pre_set_header():
+    replog = tox.result.ResultLog()
```

Comment:
again an attribute access is needed because of a module import instead the class/function import